.dotnet/src/Custom/Moderations/ModerationFlaggedContentModalities.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace OpenAI.Moderations;
+
+[Flags]
+public enum ModerationFlaggedContentModalities : int
```

Comment:
When we add support for multimodal inputs, we should confirm that, contrary to the examples in OpenAI's documentation, these flags are only set when the `Flagged` property of `ModerationResult` is `true`. Otherwise, having the word "Flagged" as part of this enum's name would be misleading.

Reply:
Given the spec and your earlier suggestion, I'll just back this up to `ModerationApplicableInputKinds`.